src/main/src/main/java/org/geoserver/catalog/LayerGroupInfo.java

```diff
@@ -88,6 +88,19 @@ public Integer getCode() {
     void setMode( Mode mode );    
 
     /**
+     * Get whether the layer group is forced to be not queryable and hence can not be subject of a GetFeatureInfo request.
+     * <p>
+     * Default is FALSE
+     * </p>
+     */
+    Boolean isNotQueryableEnabled();
```

Comment:
I would not alter current behavior. Now, a LayerGroup is queryable if some child layer is queryable (https://github.com/geoserver/geoserver/blob/master/src/wms/src/main/java/org/geoserver/wms/WMS.java#L972). Do you prefer that I change this ?